src/preamble.js

```diff
@@ -1156,6 +1156,7 @@ Module['asm'] = function(global, env, providedBuffer) {
 #if RELOCATABLE || !WASM_BACKEND
   env['__memory_base'] = {{{ GLOBAL_BASE }}}; // tell the memory segments where to place themselves
 #if WASM_BACKEND
+  env['__stack_pointer'] = STACK_BASE;
```

Comment:
To be sure I follow this, this only happens in the relocatable case - in the normal case the stack pointer is not dynamic, and the wasm has it baked in?

Reply:
Exactly